eng/common/TestResources/New-TestResources.ps1

```diff
@@ -249,6 +249,10 @@ function BuildDeploymentOutputs([string]$serviceDirectoryPrefix, [object]$azCont
         "${serviceDirectoryPrefix}AZURE_AUTHORITY_HOST" = $azContext.Environment.ActiveDirectoryAuthority;
         "${serviceDirectoryPrefix}RESOURCE_MANAGER_URL" = $azContext.Environment.ResourceManagerUrl;
         "${serviceDirectoryPrefix}SERVICE_MANAGEMENT_URL" = $azContext.Environment.ServiceManagementUrl;
+        "AZURE_CLIENT_ID" = $TestApplicationId;
+        "AZURE_CLIENT_SECRET" = $TestApplicationSecret;
+        "AZURE_TENANT_ID" = $azContext.Tenant.Id;
+        "AZURE_SUBSCRIPTION_ID" =  $azContext.Subscription.Id;
```

Comment:
At least in .NET, it was intentional that we stopped doing this because, if you work on multiple SDKs, they could conflict. Thus, we create test credentials like so: https://github.com/Azure/azure-sdk-for-net/blob/b17df4a9a8802bd535dcf4a070b548224d5d78b7/sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs#L195-L223

I think all languages doing something similar should work this way instead lest there be conflicts working on multiple SDKs - at least within the same host process (VS, terminal, etc.).